code/modules/materials/_materials.dm

```diff
@@ -77,6 +77,9 @@
 	var/mechanics_text
 	var/antag_text
 
+	var/toxicity = 0 // Organ damage from ingestion.
+	var/toxicity_targets_organ // Bypass liver/kidneys when ingested, harm this organ directly.
```

Comment:
Helpful to indicate what class of values this should be set to, as there are a bunch of ways to refer to body parts.